dataset/src/main/kotlin/com/cosmotech/dataset/bulk/PropertyType.kt

```diff
@@ -26,7 +26,7 @@ fun Any?.toPropertyType(): PropertyType {
         "true",
         "false" -> type = PropertyType.BI_BOOL
         else -> {
-          if (this.startsWith('[') and this.endsWith(']')) type = PropertyType.BI_ARRAY
+          if (this.startsWith('[') and this.endsWith(']')) type = PropertyType.BI_STRING
```

Comment:
Add a comment to explain that's a workaround and may not be fix as rRdis graph is decommissioned